README.md

```diff
@@ -36,6 +36,31 @@ projects. This tool does not only detect them, but also fixes them for you.
 
 ## Documentation
 
+### Run with docker
```

Comment:
pls put it after `### Installation` , I would still expect ppl to install first via Composer

Reply:
Ah, good idea 👍. Also `Docker` with ucfirst, I missed it somehow 😅.

Reply:
Fixed it.